crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgr.cs

```diff
@@ -14,7 +15,11 @@ public abstract class LodDataMgr
     {
         public abstract string SimName { get; }
 
-        public abstract RenderTextureFormat TextureFormat { get; }
+        // This is the texture format we want to use.
+        public abstract GraphicsFormat TextureFormat { get; }
```

Comment:
It may be an easy trap to fall into to use this desired/requested format in future code which could generate tricky bugs. Could it be marked as potentially incorrect - maybe call it 'RequestedGraphicsFormat' or something? Could it be made protected instead of public?

Reply:
Sure. Done both.